manager-bundle/bin/contao-api

```diff
@@ -28,5 +29,10 @@ if (file_exists(__DIR__.'/../../../../autoload.php')) {
 /** @var Composer\Autoload\ClassLoader $loader */
 $loader = require $projectDir.'/vendor/autoload.php';
 
+// Automatically disable xdebug if enabled. You can enable it running "CONTAO_API_ALLOW_XDEBUG=1 vendor/bin/contao-api"
+$xdebug = new XdebugHandler('CONTAO_API');
+$xdebug->check();
+unset($xdebug);
```

Comment:
```suggestion
(new XdebugHandler('CONTAO_API'))->check();
```

Reply:
It's copied from the docs. Not sure but maybe having memory freed explicitly there has a reason ;)

Reply:
If you do not set the variable, you do not have to free anything at all. 🤷🏻‍♂️

Reply:
`unset()` triggers direct gc in PHP - therefore you also should pass all variables when unsetting multiple variables.